docs/only-allow-admins-to-create-new-streams-feature.md

```diff
@@ -0,0 +1,12 @@
+# Only allow admins to create new streams
+
```

Comment:
Can you add a brief summary? One sentence is find, and if you need a reference check the other docs. It should mention that this option can only be enabled by administrators and that regular users won't be able to create streams if that option is enabled.